src/store.coffee

```diff
@@ -16,7 +16,8 @@ module.exports = (session) ->
 		###
 		
 		constructor: (config, options) ->
-			@table = options.table if options?.table
+			@table = "[" + options.table + "]" if options?.table
+			@table = "[" + config.schema + "].#{@table}" if config?.schema
```

Comment:
Yeah that works.  Would need to add square brackets  or require the user to ensure square brackets are applied if the schema or table has spaces / special characters.